docs/documentation/maintenance-coordinator.md

```diff
@@ -0,0 +1,91 @@
+# Coordinator Model
+
+## Table of Contents
+
+<!-- MarkdownTOC depth=0 autolink="true" bracket="round" autoanchor="true" style="ordered" indent="   " -->
+1. [Role of the Coordinator](#role-of-a-coordinator)
+1. [Change Management](#change-management)
+1. [Issue Management](#issue-management)
+1. [Scenarios](#scenarios)
+1. [What can be expected of a coordinator?](#expectations)
+
+
+<!-- /MarkdownTOC -->
+<a name="role-of-a-coordinator"></a>
+## Role of the Coordinator
+
+The Coordinator is not the owner of the component but much more the curator. 
+The coordinator ensures the quality of contributions to the component 
+and makes it possible for others to commit. The coordinator is responsible 
+that the documentation is kept up to date by the contributors and that the 
+guidelines of the component are met. The coordinator moderates the discussion on 
+finding a vision and on the development. The coordinator further is contact 
+for any sort of question that may arise about the respective component.
+
+The motivation of the coordinator is mostly driven by the need of a 
+reliable component for a certain aspect. Further the coordinator is 
+probably the most attractive contractor for clients aiming to change 
+aspects of the component due to the very indepth know how and the 
+listing as coordinator.
+
+<a name="change-management"></a>
+## Change Management
+Everybody may contribute to any aspect of the component. Such contributions 
+are handed in by pull requests or some other source of data if declared so in the components guidelines. 
+Pull requests on the public interface must be accepted by the JF. The coordinator gives a 
+recommendation to the JF on whether to accept or decline the PR. The decision of the JF may 
+be implicit if no objections to the recommendation of the coordinator is made. If no agreement 
+is achieved in the JF, the Technical Board will decide upon the request.
+
+Final implementations without further changes on the interface do not need 
+formal approval by the JF. The merge of the implementation is performed 
+by the coordinator or the coordinator may assign somebody to do so.
+
+Note that the general process for feature requests must be respected. However,
+this process is currently under review. The respective document will be linked as soon
+as available.
+
+<a name="issue-management"></a>
+## Issue Management
+Everybody is invited to make proposals on how to tackle any issue by proposing 
+a respective PR. Issues of the component must be reported as bugs. The coordinators 
+are responsible to assign the developer in charge on solving the bug. Due to the focus 
+on code quality, a low amount of bugs should be expected.
```

Comment:
How will the focus on code quality be assured? Will the branches set as [protected](https://help.github.com/articles/about-protected-branches/)?

Reply:
In the end there is no assurance, we somehow need to trust the coordinator that he pursues that goal and treats contributions accordingly. Technical measures won't be able to guarantee anything. This still makes the question about demotion more interesting, though. Also I'm not against protected branches or any technical measures that makes coordinators, contributors or anyones lives easier, but to me this is not about guarantees.

Reply:
Through detailed discussion of the interfaces beforehand and through review of contributions through code quality. Since multiple people will work on the component, we expect the coordinator to introduce further measures such the requirement of unit tests.